pkg/daemon/ceph/client/command.go

```diff
@@ -53,6 +53,8 @@ const (
 	CommandProxyInitContainerName = "cmd-proxy"
 	// ProxyAppLabel is the label used to identify the proxy container
 	ProxyAppLabel = "rook-ceph-mgr"
+	// DefaultAutoscaleMode means the default autoscale mode of pg
+	DefaultAutoscaleMode = "on"
```

Comment:
It seems to me that the default autoscale mode should be `""` so that we use Ceph's default mode rather than specifying a default mode in Rook.